sdk/src/nonce_instruction.rs

```diff
@@ -119,6 +132,14 @@ pub fn withdraw(nonce_pubkey: &Pubkey, to_pubkey: &Pubkey, lamports: u64) -> Ins
     )
 }
 
+pub fn authorize(nonce_pubkey: &Pubkey, authorized_noncer: &Pubkey) -> Instruction {
+    Instruction::new(
+        id(),
+        &NonceInstruction::Authorize(*authorized_noncer),
+        vec![AccountMeta::new(*nonce_pubkey, true)],
```

Comment:
@rob-solana the account key does not in fact need to sign (and may be unable to in the case of a derived address) here, correct?  Same for `Nonce` instruction above?

Reply:
Actually looks like more plumbing will be needed for this.  There doesn't seem to be a way to add a signer that isn't specified in an `AccountMeta` of one of the instructions

Reply:
go copy the code that stake and vote use for building the account metas